recipes/splipy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "splipy" %}
+{% set version = "1.5.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Splipy-{{ version }}.tar.gz
+  sha256: 6b204494fc7b4cc5c70d4ffa5db1cf20fd5326e23aa6493c743912b148b5ca22
+  path: PyPI_text.md
```

Comment:
If you just want to include the file in the recipe, you can just commit the file as you've done here. You don't actually need to list it.
```suggestion
```

Reply:
Ok thanks.

Reply:
still no luck, looks like the PyPI_text.md is not getting added to the right directory?

    FileNotFoundError: [Errno 2] No such file or directory: 'PyPI_text.md'

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=354069&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=627